### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by madneal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ This project follows the [all-contributors](https://github.com/all-contributors/
 [toc-community]: https://github.com/RustScan/RustScan#-community "Community"
 [links-table-1]: https://github.com/RustScan/RustScan#-full-installation-guide "Full installation guide"
 [links-table-2]: https://rustscan.github.io/RustScan/ "Rustscan"
-[discord]: http://discord.skerritt.blog "Discord blog"
+[discord]: https://discord.skerritt.blog "Discord blog"
```
Comment (bee-san) [inline README.md:213]: SSL does not exist for this domain

File: src/input.rs [hunk 2]
```diff
@@ -70,7 +70,7 @@ fn parse_range(input: &str) -> Result<PortRange, String> {
 /// Fast Port Scanner built in Rust.
 /// WARNING Do not use this program against sensitive infrastructure since the
 /// specified server may not be able to handle this many socket connections at once.
-/// - Discord  <http://discord.skerritt.blog>
+/// - Discord  <https://discord.skerritt.blog>
```
Comment (bee-san) [inline src/input.rs:73]: same

File: src/main.rs [hunk 3]
```diff
@@ -209,7 +209,7 @@ fn print_opening(opts: &Opts) {
     let info = format!(
         "{}\n{}\n{}\n{}",
         r#"________________________________________"#,
-        r#": http://discord.skerritt.blog         :"#,
+        r#": https://discord.skerritt.blog         :"#,
```
Comment (bee-san) [inline src/main.rs:212]: same

**Review by bee-san — CHANGES_REQUESTED** [review]
some few comments, thanks for ur contrib :)

File: src/main.rs [hunk 4]
```diff
@@ -81,10 +81,10 @@ fn main() {
     let batch_size: u16 = AVERAGE_BATCH_SIZE;
 
     // Added by wasuaje - 01/26/2024:
-    // exclude_ports  is an exclusion port list
+    // exclude_ports is an exclusion port list
     //
     // Added by brendanglancy - 5/19/2024:
-    // udp is an option to do a udp scan
+    // udp is an option to do an udp scan
```
Comment (PsypherPunk) [inline src/main.rs:87]: All the other changes make sense but my take is that this is already correct: if this is pronounced "yoo-dee-pee", it should use "a" as an article.

**Pull request closed without merging**